morf-h2/src/main/java/org/alfasoftware/morf/jdbc/h2/H2.java

```diff
@@ -82,12 +86,7 @@ public String formatJdbcUrl(JdbcUrlElements jdbcUrlElements) {
     // The DEFAULT_LOCK_TIMEOUT=60000 sets the default lock timeout to 60
     //    seconds. When the value is not set, it takes default
     //    org.h2.engine.Constants.INITIAL_LOCK_TIMEOUT=2000 value
-    // The LOB_TIMEOUT defines how long a lob returned from a ResultSet is available post-commit, defaulting to 5 minutes (300000 ms)
-    //    Set this to 2 seconds to allow certain tests using lob fields to work
-    // The MV_STORE is a flag that governs whether to use the new storage engine (defaulting to true as of H2 version 1.4, false in prior versions)
-    //  Note that implementations of H2 prior to version 1.4.199 had an MVCC parameter used to allow higher concurrency.
-    //  This configuration has been removed and the old "PageStore" implementation (MV_STORE=FALSE) is no longer supported.
-    builder.append(";DB_CLOSE_DELAY=-1;DEFAULT_LOCK_TIMEOUT=60000;LOB_TIMEOUT=2000;MV_STORE=TRUE");
+    builder.append(";DB_CLOSE_DELAY=-1;DEFAULT_LOCK_TIMEOUT=60000");
 
```

Comment:
Is there a reason for removing the LOB_TIMEOUT ? It has caused problems in the past particularly for integration tests that reset the start position frequently with a large number of business rules leading to a large memory footprint

Reply:
There has been some rebasing, I assume https://github.com/alfasoftware/morf/pull/98/files is now the best summary of the actual changes going in..